ddtrace.gemspec

```diff
@@ -55,5 +55,8 @@ Gem::Specification.new do |spec|
   # Used by appsec
   spec.add_dependency 'libddwaf', '~> 1.3.0.1.0'
 
+  # Used by profiling
+  spec.add_dependency 'libddprof', '~> 0.6.0.1.0'
```

Comment:
It's real now!

Reply:
![](https://media.giphy.com/media/12va1PkLHzeLBK/giphy.gif)

LET'S GO!!!